### PR TITLE
1561 update cookie warning stvi

### PR DESCRIPTION
**Pull request by stanvins**

### CODE REVIEW

**Comment by stanvins** [comment]
I tracked down the source of this integration error (in FilterPage test). I don't know when it worked last, but it isn't working now, and it has nothing to do with this pull request. Should I try to fix it and include it in this pull request? or make a new issue for it?

File: src/client/components/base/Head/Head.js [hunk 1]
```diff
@@ -76,7 +80,8 @@ const Head = ({
       {(og && og.book && buildOGMeta(og.book)) || null}
 
       <script>
```
Comment (kfudbc) [inline src/client/components/base/Head/Head.js:82]: Måske bedre at rykke trackingApproved ud foran <script>:
{trackingApproved && <script> ...} 
så vi ikke får et tomt script

Comment (stanvins) [reply src/client/components/base/Head/Head.js:82]: ok. Ill see what I can do.

Comment (stanvins) [reply src/client/components/base/Head/Head.js]: done

**Pull request merged**